src/instance.ts

```diff
@@ -41,6 +61,15 @@ import {Database} from './database';
  * const instance = spanner.instance('my-instance');
  */
 class Instance extends common.ServiceObject {
+  // tslint:disable-next-line: no-any
+  metadata: any;
```

Comment:
I think you can pull in `Metadata` from the common module.

Reply:
taken care in Instance PR#619